single-audit/single_audit/urls.py

```diff
@@ -17,10 +17,12 @@
 # from fac import views  # @todo: Revisit, split out by app.
 from fac.views import get_single_audit_package
 from distiller import views
+from resolve_findings.views import finding_resolution_page
```

Comment:
I'm good with this; will split out by app as part of a separate PR. (Or in #101.)

Reply:
👍